reservations/models.py

```diff
@@ -22,6 +22,11 @@
 User = get_user_model()
 
 
+class ReservationType(models.TextChoices):
+    TYPE_NORMAL = "normal"
```

Comment:
Could these be called as `NORMAL` and `BLOCKED`?

Reply:
Sure! Fixed 🛠️